modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/components/PartialResults.scss

```
@@ -31,6 +36,10 @@
 		padding-top: 24px;
 	}
 
+	&--hide-locale-actions {
+		display: none;
+	}
```

I think you don't need create this class here, because the clase `hide` was already defined in Forms.
```suggestion
	&--hide-locale-actions {
		display: none;
	}
```

E.g: `modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/Report.es.js`